snuba/admin/static/capacity_management/types.tsx

```diff
@@ -1,31 +1,41 @@
 import { ReactNode } from "react";
 
-type AllocationPolicy = {
-  policy_name: string;
-  configs: AllocationPolicyConfig[];
-  optional_config_definitions: AllocationPolicyOptionalConfigDefinition[];
+
+interface ConfigurableComponent {
+  type: string;
+  name: string;
+  configs: Configuration[];
+  optional_config_definitions: OptionalConfigurationDefinition[];
+};
```

Comment:
if you're making a more generic thing, it should not be housed under capacity management